ksp_plugin/vessel.cpp

```diff
@@ -577,11 +577,10 @@ void Vessel::StartPrognosticatorIfNeeded() {
 }
 
 Status Vessel::RepeatedlyFlowPrognostication() {
-  for (;;) {
-    // No point in going faster than 50 Hz.
-    std::chrono::steady_clock::time_point const wakeup_time =
-        std::chrono::steady_clock::now() + std::chrono::milliseconds(20);
-
+  // No point in going faster than 50 Hz.
+  for (auto wakeup_time = std::chrono::steady_clock::now();;
+       wakeup_time += std::chrono::milliseconds(20),
+            std::this_thread::sleep_until(wakeup_time)) {
```

Comment:
Nit: the indent is weird.  Probably move left by 1 position.

Reply:
It now looks a bit different since the wakeup_time computation has been moved into the loop body. Does it still look weird? FYI I'm using clang_format for formatting.